src/ui/public/config/defaults.js

```diff
@@ -200,7 +200,11 @@ define(function (require) {
       },
       'dashboard:defaultDarkTheme': {
         value: false,
-        description: 'New dashboards use dark theme by default',
+        description: 'New dashboards use dark theme by default'
+      },
+      'filters:defaultGlobalState': {
```

Comment:
I appreciate that this is true to what is happening, but it feels a bit like leaking implementation details. I think it would be better named `filterbar:pinnedByDefault`, or something else that defines the desired effect rather than implementation.